src/ripple/app/misc/SHAMapStoreImp.cpp

```diff
@@ -171,7 +171,7 @@ SHAMapStoreImp::SHAMapStoreImp(
     , scheduler_ (scheduler)
     , journal_ (journal)
     , working_(true)
-    , canDelete_ (std::numeric_limits <LedgerIndex>::max())
+    , canDelete_ (0)
```

Comment:
This change would seem to affect the behavior of this bit of code: https://github.com/ripple/rippled/blob/develop/src/ripple/app/misc/SHAMapStoreImp.cpp#L368

Is the behavior of that bit of code still correct?

Reply:
Yes, it's still correct. If advisory_delete is enabled, that value is set to 0 initially. The only reason that it was set to std::numeric_limits <LedgerIndex>::max() originally was to be fed to shouldAcquire() that way if advisory_delete was not set.